devolutions-gateway/src/api/jrec.rs

```diff
@@ -179,44 +193,32 @@ async fn jrec_delete_many(
     }): State<DgwState>,
     _scope: RecordingDeleteScope,
     Json(delete_list): Json<Vec<Uuid>>,
-) -> Result<(), HttpError> {
-    // When deleting many many recordings, this synchronous block may take more than 250ms to execute.
-    let join_handle = tokio::task::spawn_blocking(move || {
-        let active_recordings = recordings.active_recordings.cloned();
-
-        let conflict = delete_list.iter().any(|id| active_recordings.contains(id));
-
-        if conflict {
-            return Err(HttpErrorBuilder::new(StatusCode::CONFLICT)
-                .msg("attempted to delete a recording for an ongoing session"));
-        }
-
-        let conf = conf_handle.get_conf();
-
-        let recording_paths: Vec<Utf8PathBuf> = delete_list
-            .iter()
-            .map(|session_id| conf.recording_path.join(session_id.to_string()))
-            .collect();
-
-        for (path, session_id) in recording_paths.iter().zip(delete_list.iter()) {
-            if !path.exists() {
-                return Err(HttpError::not_found()
-                    .with_msg("attempted to delete a recording not found on this instance")
-                    .err()(format!(
-                    "folder {path} for session {session_id} does not exist"
-                )));
-            }
-        }
-
-        Ok((delete_list, recording_paths))
-    });
-
-    let (delete_list, recording_paths) = join_handle.await.map_err(HttpError::internal().err())??;
+) -> Result<Json<DeleteManyResult>, HttpError> {
+    use std::collections::HashSet;
+
+    const BLOCKING_THRESHOLD: usize = 100_000;
```

Comment:
[qeustion]: This threshold looks pretty big, what is the measured delay for processing 100000 items without `spawn_blocking`?